identity/README.md

```diff
@@ -31,7 +31,7 @@ So the required steps are:
 
 * configure and run nginx (see `/nginx`)
 * update your hosts file (also described in `/nginx`)
-* make sure that your frontend.properties file contains
+* make sure that your `~/.gu/frontend.conf` file contains
```

Comment:
no this should be removed completely, if it doesn't work with the default settings they should be added to the master file in s3.  The local file is only for user specific overrides.